pkg/alertmanager/alertmanager.go

```diff
@@ -359,17 +359,21 @@ func (am *Alertmanager) ApplyConfig(userID string, conf *config.Config, rawCfg s
 		return nil
 	}
 
-	muteTimes := make(map[string][]timeinterval.TimeInterval, len(conf.MuteTimeIntervals))
+	timeIntervals := make(map[string][]timeinterval.TimeInterval, len(conf.MuteTimeIntervals)+len(conf.TimeIntervals))
```

Comment:
I'm not sure if this warrants a test - I can't seem to find any precedence for such; any ideas?

Reply:
Ideally yes, but I wouldn't block on it, given the muted times weren't tested even before.

BTW, thanks for fixing this!

Reply:
Sounds good, I filed https://github.com/grafana/mimir/issues/3100 to follow up on it.